.idea/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Default ignored files
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources.local.xml
```

Comment:
from the pull request. This can be done in two ways:
1) Amend the last commit and remove the file from there.
2) Create a new commit where the file will be removed.

I would recommend choosing the first method considering the small changes.